docs/source/markdown/options/expose.md

```diff
@@ -2,7 +2,11 @@
 ####>   podman create, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
-#### **--expose**=*port*
+#### **--expose**=*port[/protocol]*
 
-Expose a port, or a range of ports (e.g. **--expose=3300-3310**) to set up port redirection
-on the host system.
+Expose a port or a range of ports (e.g. **--expose=3300-3310**).
```

Comment:
As someone who rarely uses this option, I might find it helpful to see something like "...to the container" or "...from the host to the container" because I can never remember which direction is which.

Reply:
There is no direction here. It is just the container port. Really I think this option should not be used unless you know exactly what you are doing. I think saying "expose" is likely the most confusing part. It doesn't expose anything unless -P is set. It is really more of a documentation option, i.e. it matches 1 to 1 to the `EXPOSE` instruction in the Dockerfile which also does nothing other than indicating that there is a service configured to listen on that port in the container.
It will also commit this info to the image config with `podman commit` so I guess that might reason to use it. 

For actual port forwarding -p/-P must be used as stated in the sentence after this.